### DISCUSSION TITLE
clipping non-centered image areas

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#395] [Q&A] [answered]
This comes from #394.

Besides the issue described there, how could any on the four corners of an image be clipped at all?

I mean, clipping seems to start from the image center (both vertically and horizontally) and there is no way to set an offset (for either axis).

Imagine a PDF image (148.5mm × 210mm) from which I want to clip an image area of 2cm wide and 3cm long from the upper left corner (or any other corner).

I guess I have to be missing something, since not all clipping may come from the center of the image.

**Comment by pgundlach** [comment, marked as answer]
There is no custom clipping. You can try to play with the following:

```
<PlaceObject>
    <Frame rulewidth="0pt">
      <Transformation matrix=" 2 0 0 2 0 0 " origin-x="50" origin-y="0">
          <Image width="5cm" file="cow.pdf"></Image>
      </Transformation>
    </Frame>
</PlaceObject>
```

See https://doc.speedata.de/publisher/en/basics/outputtingobjects/index.html#ch-outputtingobjects-transformation

**Reply by pr-apes** [reply]
Sorry, I have no math background. This is really all Greek to me.
